### PR TITLE
add example rbac policy in permissive mode

### PR DESCRIPTION
**Pull request by quanjielin**
add samples yaml file used in [istoi.io example](https://github.com/istio/istio.github.io/pull/2426)

### CODE REVIEW

**Comment by quanjielin** [comment]
the unit test fail isn't caused by this PR, all recent test fail

File: samples/bookinfo/platform/kube/rbac/rbac-config-ON-permissive.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: "rbac.istio.io/v1alpha1"
```
Comment (diemtvu) [inline samples/bookinfo/platform/kube/rbac/rbac-config-ON-permissive.yaml]: let's use lowercase for filename.

File: samples/bookinfo/platform/kube/rbac/rbac-permissive-telemetry.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Configuration for logentry instances
+apiVersion: "config.istio.io/v1alpha2"
+kind: logentry
+metadata:
+  name: newlog
```
Comment (diemtvu) [inline samples/bookinfo/platform/kube/rbac/rbac-permissive-telemetry.yaml]: can we use more specific name. Here and others

Comment (quanjielin) [reply samples/bookinfo/platform/kube/rbac/rbac-permissive-telemetry.yaml]: updated.

**Comment by costinm** [comment]
For any change that have the potential to affect the stability ( policy server crashes are cascading and have huge impact on latency ) - please add the results of the scale testing - i.e. what is the latency and mixer and envoy CPU/memory usage before and after ( or with the change on and off ).

I think this must be done for any PR related to policy checks, it's too critical.

**Comment by quanjielin** [comment]
@costinm @mandarjog 
FYI, https://github.com/istio/istio/issues/5345, https://github.com/envoyproxy/envoy/pull/3554 are the main bug for more background

**Pull request merged**